front-end-ui/README.md

```diff
@@ -1,3 +1,143 @@
-# FrontEndUi
+# Using Acme Gifts
+
+After having gone through the setup process, you should now be at landing page (by default: https://localhost:5064). 
```

Comment:
I think we should link to the setup process, I think you're referring to the main README.md.

Reply:
good point